### ISSUE TITLE
Forcestack entity amount message incorrect (low priority)

### ISSUE THREAD

**Issue by andrewkm** [Nathat23/StackMob-5#262]
Paper 1.19.2 Build 142
StackMob-5 built upon: https://github.com/Nathat23/StackMob-5/commit/e5b74af5308928a8135ff1d97d4dc6bf1d236b66 (latest)

I have no idea how to explain this (sorry for typing up this bug report in a rush) but like, the msg in chat/console isn't correct, check this out: https://www.youtube.com/watch?v=ZgdSbhSdP2Y

**Comment by andrewkm** [comment]
I realize the video demonstration here is total shit.
I'll work on a better one shortly, just finishing up another bug report.

**Comment by andrewkm** [comment]
Updated video:
https://youtu.be/GK8OQiB3OOI

**Comment by andrewkm** [comment]
Updated video https://youtu.be/vQuSiFNc3O0
Tested against https://github.com/Nathat23/StackMob-5/commit/7e2dd33407d0c7b7dde411bd5ff87312b5744325 & https://github.com/dmulloy2/ProtocolLib/commit/389233f599b6d0008e607f8c498bfb152a130424

@Nathat23 can me maybe, perhaps, possible, hopefully, lol, check this out finally xD

**Comment by Nathat23** [comment]
Just checking... so if you used /spawnmob and waited they don't stack? It's only after '/sm forcestack' is ran that they stack.

**Comment by andrewkm** [comment]
If I use /spawnmob and wait they correctly stack no problem.
**sm forcestack** doesn't stack anything as you can see in the video. It's literally a broken command.
